.test-infra/tools/stale_dataflow_prebuilt_image_cleaner.sh

```diff
@@ -89,24 +98,24 @@ for image_name in ${IMAGE_NAMES[@]}; do
           MANIFEST=$(docker manifest inspect ${image_name}@"${current}" || echo "")
           if [ -z "$MANIFEST" ]; then
             # Sometimes "no such manifest" seen. Skip current if command hit error
-            FAILED_INSPECT+=" $current"
+            FAILED_IMAGES+=" $current"
             continue
           fi
           SHOULD_DELETE=0
           DIGEST=$(echo $MANIFEST |  jq -r '.manifests[0].digest')
           if [ "$DIGEST" != "null" ]; then
             SHOULD_DELETE=1
-            for i in ${STALE_IMAGES_CURRENT[@]}; do
-              if [ "$i" = "$DIGEST" ]; then
+            for (( j_stale_images_current=0; j_stale_images_current<${#STALE_IMAGES_CURRENT[@]} ; j_stale_images_current+=2 )) ; do
```

Comment:
Why ware we double incrementing here?

Reply:
Now STALE_IMAGES_CURRENT is of format (digest creation_year digest creation_year ...) so we do not need to make a second  docker API call (`docker inspect ...`) for the images of nonzero length (actual image)

Reply:
Ah, I see - got it, thanks